src/components/Cards/Recipe-Card/Recipe-Card.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+
+export default function RecipeCard(props) {
+  return (
+    <div className="max-w-sm rounded overflow-hidden shadow-lg">
+      <img
+        className="w-full"
+        src={require('./img/breakfast3.png')}
+        alt="breakfast"
+      />
+      <div></div>
+      <div className="px-6 py-4 flex flex-col justify-around items-center ">
+        <h1 className="text-black-700 text-base font-bold text-xl mb-2 text-center">
+          Kiwi, Greek yogurt <br />& blueberries
+        </h1>
+        <button
+          className="bg-primary text-white active:bg-pink-600 font-poppins  text-sm px-6 py-3 rounded shadow hover:shadow-lg outline-none focus:outline-none mx-4"
+          type="button"
+          style={{ transition: 'all .15s ease' }}
+        >
+          See Recipe
+        </button>
+
+        <div className="my-4 flex flex-wrap justify-around items-center">
+          <div className=" flex flex-col justify-around items-center w-1/3">
```

Comment:
`justify-center` is better here than `justify-around` we usually want our items to be centered in their containers 
```suggestion
          <div className=" flex flex-col justify-center items-center w-1/3">
```